java/code/src/com/redhat/rhn/domain/scc/SCCCachingFactory.java

```diff
@@ -220,7 +220,7 @@ public static List<SCCOrderItem> listOrderItemsByCredentials(ContentSyncSource s
                                     SELECT * from suseSCCOrderItem
                                     WHERE credentials_id = :credentials
                                     """, SCCOrderItem.class)
-                            .setParameter("credentials", credentials, StandardBasicTypes.LONG)
```

Comment:
Is it really necessary? As far as I understand, specifying the type helps prevent many potential issues I encountered during the Hibernate upgrade. What do you think about something like this?
```
.setParameter("credentials", credentials.getId(), StandardBasicTypes.LONG)
```